runtime/jcl/common/sun_misc_Unsafe.cpp

```diff
@@ -1101,7 +1101,7 @@ Java_jdk_internal_misc_Unsafe_registerNatives(JNIEnv *env, jclass clazz)
 	if (J2SE_VERSION(currentThread->javaVM) >= J2SE_V11) {
```

Comment:
I suggest this condition should be updated as well:
```
	if (JAVA_SPEC_VERSION >= 10) {
```

Reply:
Sure, I think you meant `if (JAVA_SPEC_VERSION >= 11)` instead.

Reply:
No, I meant `10`; see the next line:
```
	registerJdkInternalMiscUnsafeNativesJava10(env, clazz);
```

Reply:
alright, updated.